workshop.html

```diff
@@ -0,0 +1,530 @@
+
+            </section>
+          </section>
+
+          <section class="eachWS halfDay">
+            <section class="eachWSTop halfTop">
+              <span>Aug 15</span>
+              <span>Half Day</span>
+            </section>
+            <section class="eachWSTitle">
+              <p><a href="" target="_blank">ANDEA: Anomaly and Novelty Detection, Explanation, and Accommodation</a></p>
+            </section>
+            <section class="eachWSOrg">
+              <p>Guansong Pang <span class = "WScompany">Singapore Management University</span></p><p> Jundong Li <span class = "WScompany">University of Virginia</span></p><p> Anton van den Hengel <span class = "WScompany">University of Adelaide</span></p><p> Longbing Cao <span class = "WScompany">University of Technology Sydney</span></p><p> Thomas G. Dietterich <span class = "WScompany">Oregon State University</span></p>
+            </section>
+          </section>
+
+          <section class="eachWS halfDay">
+            <section class="eachWSTop halfTop">
+              <span>Aug 15</span>
+              <span>Half Day</span>
+            </section>
+            <section class="eachWSTitle">
+              <p><a href="https://charliezhaoyinpeng.github.io/EAI-KDD22/cfp/" target="_blank">1st ACM SIGKDD Workshop on Ethical Artificial Intelligence: Methods and Applications (EAI-KDD’22)</a></p>
+            </section>
+            <section class="eachWSOrg">
+              <p>Chen Zhao <span class = "WScompany">Kitware Inc.</span></p><p> Feng Chen <span class = "WScompany">UT Dallas</span></p><p> Xintao Wu <span class = "WScompany">University of Arkansas</span></p><p> Christopher Funk <span class = "WScompany">Kitware, Inc</span></p><p> Anthony Hoogs <span class = "WScompany">Kitware</span></p>
+
+            </section>
+          </section>
+
+        </section>
+        
+      </section>
+
+
+    <footer id="commonFooter">
+    <!-- load by jquery-->
+    </footer>
+
+    <!-- SCRIPTS -->
+    <script src="https://code.jquery.com/jquery-3.2.1.min.js" integrity="sha256-hwg4gsxgFZhOsEEamdOYGBf13FyQuiTwlAQgxVSNgt4=" crossorigin="anonymous"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.12.9/umd/popper.min.js" integrity="sha384-ApNbgh9B+Y1QKtv3Rn7W3mgPxhU9K/ScQsAP7hUibX39j7fakFPskvXusvfa0b4Q" crossorigin="anonymous"></script>
+    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/js/bootstrap.min.js" integrity="sha384-JZR6Spejh4U02d8jOt6vLEHfe/JQGiRRSQQxSfFWpi1MquVdAyjUar5+76PVCmYl" crossorigin="anonymous"></script>
+    <script>
+    $(function(){
+      $("#commonNavBar").load("navBar.html"</span></p><p>
+      $("#commonFooter").load("footer.html"</span></p><p>
+    }</span></p><p>
```

Comment:
Is this a typo here?
`</span></p><p>` on the tail of Line 524, 525. 
I could not see navBar loaded.